modules/publicize/publicize-jetpack.php

```diff
@@ -625,8 +604,14 @@ function options_page_facebook() {
 
 					</tbody>
 				</table>
-
-			<?php endif; ?>
+			<?php else: ?>
+				<div>
+					<p><?php _e( sprintf( 'Only Facebook pages are now supported by Publicize. <a href="%s">Learn more about this change</a>', 'https://jetpack.com/support/publicize/facebook' ), 'jetpack' ); ?></p>
```

Comment:
I think we should include this note in the page selection dialog as well as here in the dialog when there are no pages.

Also, the text here makes it sound like Publicize is limited (when really it's Facebook). If I saw the current message, I would probably look for a different plugin to automate posting to my Facebook Profile (not realizing it was no longer possible).

I don't want to reword it to blame Facebook and make us look better :) but making it clear that this is limitation of Facebook's will help people understand the situation better.

Maybe something like the following?

> Facebook only supports Publicize for Facebook Pages. Publicize for Facebook Profiles is not possible. Learn More about Publicize for Facebook.

I'm sure +editorial will be able to help as well.

Reply:
This i18n is broken. `_e()` can't contain `sprintf()`. `printf( __() )` will work better.